input/mapping/BFDR_Profile_Intros.csv

```diff
@@ -129,19 +129,19 @@ See [Patient Worksheet Questionnaires](patient_worksheet_questionnaires.html) fo
-101,Input,ObservationNumberPreviousCesareans,Observation-number-previous-cesareans,The edit flag extension supports validation as part of the Jurisdiction to NCHS use case and can be ingnored for the provider to jurisdiction use case.  The validation checks  are done at the jurisdiction prior to sending to the National Statistical Agency and are based on the item specific edit criteria specified in the Birth Edit Specifications for the 2003 Proposed Revision of the U.S. Standard Certificate of Birth and the Fetal Death Edit Specifications for the 2003 Revision of the U.S. Standard Report of Fetal Death.,x,x,BFDR
-102,Input,ObservationEducationLevelVitalRecords,Observation-education-level-vr,The edit flag extension supports validation as part of the Jurisdiction to NCHS use case and can be ingnored for the provider to jurisdiction use case.  The validation checks  are done at the jurisdiction prior to sending to the National Statistical Agency and are based on the item specific edit criteria specified in the Birth Edit Specifications for the 2003 Proposed Revision of the U.S. Standard Certificate of Birth and the Fetal Death Edit Specifications for the 2003 Revision of the U.S. Standard Report of Fetal Death.,x,x,VRCPL
-103,Input,ObservationPaternityAcknowledgementSigned,Observation-paternity-acknowledgement-signed,,x,x,BFDR
-104,Input,ObservationPlannedToDeliverAtHome,Observation-planned-to-deliver-at-home,,x,x,BFDR
-105,Input,ObservationPluralityVitalRecords,Observation-plurality-vr,"('Reabsorbed' fetuses, those which are not 'delivered' [expulsed or extracted from the mother] should not be counted.) Include all live births and fetal losses resulting from this pregnancy. 
+94,Input,ObservationNoneOfSpecifiedObstetricProcedures,Observation-none-of-specified-obstetric-procedures,"Presence of this observation indicates that none of the obstetric procedures specifyable in this IG are reported. If the none-of-the-above observation is present in the bundle, then its complement should not be used (see [note on missing obstetric procedures data])",x,x,BFDR
+95,Input,ObservationNoneOfSpecifiedInfectionsPresentDuringPregnancy,Observation-no-infections-present-during-pregnancy,"Presence of this observation indicates that none of the infections specifyable in this IG are reported. If the none-of-the-above observation is present in the bundle, then its complement should not be used (see [note on missing infections present data])",x,x,BFDR
+96,Input,ObservationNumberBirthsNowDead,Observation-number-births-now-dead,,x,x,BFDR
+97,Input,ObservationNumberBirthsNowLiving,Observation-number-births-now-living,,x,x,BFDR

```

Comment:
Curious, Why does github think these differences so big?  Why isn't it a one liner?

Reply:
Likely numbering; also could be a CRLF issue when working between mac/windows